CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
 # PD Change Log
+
+## v2.0.0-GA
+### New Features
+* Support using pd-ctl to scatter spcified Regions for manual adjust hotspot regions in some cases
+### Improves
+* More configuration check rules to prevent unreasonable configurations
+* Optimize scheduling strategy when TiKV node has insufficient space, try the best to prevent disk from becoming full
```

Comment:
Suggestion: Optimize the scheduling strategy when a TiKV node has insufficient space so as to prevent the disk from being fully occupied